hamilton/function_modifiers/macros.py

```diff
@@ -876,3 +876,134 @@ def optional_config(self) -> Dict[str, Any]:
 #
 #     def __init__(self, *transforms: Applicable, collapse=False):
 #         super(flow, self).__init__(*transforms, collapse=collapse, _chain=False)
+
+
+class post_pipe(base.NodeInjector):
+    def __init__(
+        self,
+        *transforms: Applicable,
+        namespace: NamespaceType = ...,
+        collapse=False,
+        _chain=False,
```

Comment:
I'd take away `_chain` -- it doesn't look like we ever got around to implementing `@flow` 😆

Reply:
Is `@flow` dead or still on the agenda?  If it is still good feature, shouldn't take too long to implement after we finish here and I can leave this in. Otherwise, I take it out.

Reply:
Yeah, IIRC `@flow` was specifically a procedural approach. `@pipe` without the chaining implication. Definitely a cool feature, and we should have it, but we have not focused. If you're interested then go for it, but let's get `@pipe`/`@post_pipe` working